<launch>
    <!-- Autogenerated launch file - do not hand edit -->
    <include file="{{machine_file}}" />
    {% for region in regions_dict %}
    <group ns="{{region}}">
        <node pkg="mct_image_stitcher" type="image_stitcher.py" name="image_stitcher" args="{{region}}" machine="mct_master" />
    </group>
    {% endfor %} 
</launch>

